components/Pagination/src/stories/pagination.bem.stories.mdx

```diff
@@ -0,0 +1,678 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import "../index";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS Components/Navigation/Pagination"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "IN DEVELOPMENT",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Pagination" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <nav class="denhaag-pagination">
+      <a href="#page-1" class="denhaag-pagination__link denhaag-pagination__link--arrow" rel="prev">
+        <svg
+          class="denhaag-icon"
+          width="1em"
+          height="1em"
+          viewBox="0 0 7 12"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M4.9921 10.8143C5.36382 11.1914 5.97222 11.1914 6.34393 10.8143C6.7079 10.4451 6.70822 9.8521 6.34466 9.48248L3.36315 6.45123C2.98039 6.06209 2.98039 5.43791 3.36315 5.04877L6.34466 2.01752C6.70822 1.6479 6.7079 1.05492 6.34394 0.685696C5.97222 0.308599 5.36382 0.308599 4.9921 0.685695L0.692003 5.04799C0.308224 5.43732 0.308224 6.06268 0.692003 6.45201L4.9921 10.8143Z"
+            fill="currentColor"
+          />
+        </svg>
+        <span class="denhaag-pagination__link-label">Vorige</span>
+      </a>
+      <span role="group" class="denhaag-pagination__links">
+        <a class="denhaag-pagination__link" href="#page-1">
+          1
+        </a>
+        <a class="denhaag-pagination__link" href="#page-2" rel="prev">
+          2
+        </a>
+        <a class="denhaag-pagination__link denhaag-pagination__link--current" href="#page-3" aria-current="true">
```

Comment:
I believe for pagination, its better if the `aria-current` uses `aria-current="page"`

Reply:
I based this logic on the pagination which was created for Utrecht, i'm not so familiar with aria

Reply:
@bddjong , i'll apply the `page`, at https://www.aditus.io/aria/aria-current/#aria-current-page they mention:

> aria-current="page"
Whenever you need to indicate the current page within a set of pagination links (main navigation), providing a screen reader alternative to styling the current page.